README.md

```diff
@@ -1,14 +1,29 @@
 # airborne-dsa-lite
-Data shipping script for airborne products 
+
+Data shipping script for airborne products
+
+## Local Development
+
+### Prerequisites
+
+- Python 3 installed
+- Python 3 venv installed
+- `python3 -m venv env`
+- `source env/bin/activate`
+- `python3 -m pip install --upgrade pip setuptools pyinstaller`
+
+### Running locally
 
 ## Getting started
-* Make sure you have python 3 installed
-* `pip install awscli boto3`
-* `aws configure`
-* Set the correct bucket name in `settings.py`
 
-Run: `python main.py` or `intterra-airborne-dsa.bat`
+- `source env/bin/activate`
+- `python3 -m python install -e .`
```

Comment:
Should this be `pip` instead of `python`?

Reply:
Yes, will resolve